books/tests/test_views.py

```diff
@@ -13,7 +14,7 @@ class TestHomePageView:
     @pytest.mark.django_db
     def test_anonymous(self):
         req = RequestFactory().get("/")
-        resp = views.HomePageView.as_view()(req)
+        resp = views_bookx.HomePageView.as_view()(req)
```

Comment:
Not a change request, just a tip:
for short imports you can also do:
```python
import bookx.views

#...
resp = bookx.views.HomePageView.as_view()(req)
```

But for longer names it's definitely usually convenient the way you did it!